workers/unity/Packages/com.improbable.gdk.core/Authority/AuthorityComponents.cs

```diff
@@ -2,14 +2,27 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     This ECS component denotes that this worker is authoritative over T.
+    /// </summary>
+    /// <typeparam name="T">The SpatialOS component.</typeparam>
     public struct Authoritative<T> : IComponentData where T : ISpatialComponentData
```

Comment:
We should change this asap to not be generic.

Reply:
is that to be codegenned? good idea! will be worth a ticket :)

Reply:
https://improbableio.atlassian.net/browse/UTY-1289